front/src/modules/ui/snack-bar/components/SnackBarProvider.tsx

```diff
@@ -68,6 +68,7 @@ export function SnackBarProvider({ children }: React.PropsWithChildren) {
   return (
     <>
       {children}
+      {/* TODO: Perhaps make the snackbar position dynamic i.e top, bottom, e.t.c */}
```

Comment:
The designs have the SnackBar at the bottom, current implementation always has it placed at the top